frontend/templates/contraceptives/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { ReactElement, useState } from 'react';
+import Efficacy, { EfficacyProps } from './tabs/Efficacy';
+import Effect, { EffectProps } from './tabs/Effect';
+import Overview, { OverviewProps } from './tabs/Overview';
+import Use, { UseProps } from './tabs/Use';
+import TabBar from '../TabBar';
+import Mechanism, { MechanismProps } from './tabs/Mechanism';
+import PracticalQuestions, { PracticalProps } from './tabs/PracticalQuestions';
+import AdditionalInfo, { AdditionalProps } from './tabs/AdditionalInfo';
+import styled from 'styled-components';
+import DownOutlined from '@ant-design/icons/DownOutlined';
+import { Column, Row } from './tabs/StyledComponents';
+import SvgBookmark from '../../public/bookmark.svg';
+
+// styled
+const Bookmark = styled(SvgBookmark)`
+  fill: white;
+  position: relative;
+  stroke: black;
+  top: 2px;
+`;
+
+const Container = styled.div`
+  padding: 1rem;
+`;
+
+const DownArrow = styled(DownOutlined)`
+  position: absolute;
+  bottom: 1rem;
+  left: 50%;
+`;
+
+const Header = styled(Container)`
+  background-color: #febba8;
+  display: flex;
+  flex-direction: row;
+`;
+
+const SvgCircle = styled.div`
+  background-color: white;
+  border-radius: 50%;
+  padding: 0.7rem 0.9rem 0.6rem 0.9rem;
+
+  :hover {
+    cursor: pointer;
+
+    ${Bookmark} {
+      fill: purple;
+    }
+  }
+`;
+
+const SvgCircleSecond = styled(SvgCircle)``;
+
+const SvgColumn = styled(Column)`
+  margin-left: auto;
+
+  ${SvgCircleSecond} {
+    margin-top: 0.5rem;
+  }
+`;
+
+const SvgRow = styled(Row)`
+  margin-left: auto;
+  margin-top: auto;
+
+  ${SvgCircleSecond} {
+    margin-left: 0.5rem;
+  }
+`;
+
+const Title = styled.h1`
+  margin: 0;
+`;
+
+// components
+export declare interface ContraceptivesProps {
```

Comment:
what does the 'declare' do here?

Reply:
According to my very intense StackOverflow research it doesn't really do anything so we can probably get rid of it with no error